src/components/calcite-dropdown-item/calcite-dropdown-item.tsx

```diff
@@ -77,6 +77,12 @@ export class CalciteDropdownItem {
   //
   //--------------------------------------------------------------------------
 
+  connectedCallback(): void {
+    this.selectionMode = getElementProp(this.el, "selection-mode", "single");
+    this.parentDropdownGroupEl = this.el.closest("calcite-dropdown-group");
+    if (this.selectionMode === "none") this.active = false;
```

Comment:
question: could this be written as `this.active = this.selectionMode === "none"`? Just thinking that the shorthand if statements can get messed up sometimes. Just a tiny nitpick.

Reply:
That would remove `active` on all items that weren't "none" though, right?

This line just clears `active` in DOM, if present, from "none" items since that's not valid (they can't be active). I can wrap it in brackets though so it's not single line.

Reply:
Just leave as is :)